bindata/manifests/cni-config/sriov-cni-config.yaml

```diff
@@ -7,8 +7,16 @@ metadata:
     k8s.v1.cni.cncf.io/resourceName: {{.SriovCniResourceName}}
 spec:
   config: '{
+{{- if .MetaPluginsConfigured -}}
+  "cniVersion":"0.4.0",
```

Comment:
@zshi-redhat Does sriov-cni support cni spec 0.4.0? If yes, shall we always use version 0.4.0 here?

Reply:
I think the major change of 0.4.0 spec is the cmdCheck, we have a dummy interface interface for cmdCheck in sriov-cni.
maybe we shall wait until 0.4.0 is fully supported in sriov-cni.

Reply:
The 0.4.0 version is mandatory for using a list of plugins though.
@zshi-redhat is it fine to leave it as it is (setting only when a meta plugin is added)?

Reply:
Can we set cniVersion for meta plugin only?
Just checked sriov-cni, it accepts all CNI versions as supported, I think we shall be fine running sriov-cni with version 0.4.0 although I didn't test myself. 
Is 0.4.0 the only version supported by VRF? I see in the README page, it uses 0.3.1 as an example, but in the code, it has 0.4.0.